backend/src/db.ts

```diff
@@ -0,0 +1,19 @@
+import { TypeOrmModuleOptions } from '@nestjs/typeorm';
+import { config } from 'dotenv';
+
+import { Book } from './books/book.entity';
+
+config();
+
+const { DB_PORT, DB_USERNAME, DB_PASSWORD, DB_DATABASE, DB_HOST } = process.env;
+
+export const ormOptions: TypeOrmModuleOptions = {
```

Comment:
I would suggest moving these into an own configuration file. e.g. `src/configuration/configuraiton.ts`

Reply:
Ops, I just realized I mentioned this earlier, sorry.

I might have miscommunicated here, but I meant having a generic configurations file. This will do for now though! 👍🏼